price-feeder/oracle/provider/websocket_controller.go

```diff
@@ -190,3 +240,14 @@ func (wsc *WebsocketController) reconnect() {
 	go wsc.Start()
 	telemetryWebsocketReconnect(wsc.providerName)
 }
+
+func (wsc *WebsocketController) pongHandler(appData string) error {
+	return nil
+}
+
+func (wsc *WebsocketController) pingHandler(appData string) error {
```

Comment:
Might want to add a comment

```suggestion
// pingHandler is used to respond to pings from xyz
func (wsc *WebsocketController) pingHandler(appData string) error {
```

Reply:
https://github.com/umee-network/umee/pull/1554/commits/166014a0f416f06f581659b8dea14a521c7e3d42